plugins/modules/mysql_user.py

```diff
@@ -294,7 +294,15 @@
 # password=n<_665{vS43y
 '''
 
-RETURN = '''#'''
+RETURN = r'''
+queries:
+  description: List of executed queries which modified DB's state.
```

Comment:
```suggestion
  description: List of executed queries which modified a state of a database.
```
We shouldn't use abbreviations if possible

Reply:
I copied that from `mysql_replication.py` for consistency, but agree that it could be better.  I'll fix it.